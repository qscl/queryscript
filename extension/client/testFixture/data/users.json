{"id": 1, "org_id": 1, "name": "Foo"}
{"id": 2, "org_id": 1, "name": "Bar"}
