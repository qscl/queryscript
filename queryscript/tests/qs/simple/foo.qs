import users from schema;

fn identity<R>(u R) -> R {
    u
}

identity(users);
