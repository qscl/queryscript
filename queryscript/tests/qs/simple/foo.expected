{
    "compile_errors": [
        (
            None,
            NoSuchEntry {
                path: [
                    "R",
                ],
                backtrace: None,
            },
        ),
    ],
    "decls": {
        "let identity": ∀ "R" λ {
        	u "R",
        } -> "R",
        "let users": [{
        	id Int32,
        	org_id Int32,
        	name Utf8,
        	active Boolean,
        }],
    },
    "queries": [
        Err(
            StringError {
                what: "Unknown type cannot exist at runtime (?async_slot?)",
                backtrace: None,
            },
        ),
    ],
}